test/functionalTest/cases/3162_context_providers_legacy_non_primitive/blanked_out_attr.test

```diff
@@ -199,9 +199,9 @@ Date: REGEX(.*)
     ],
     "attrs": {
         "address": {
-            "creDate": REGEX([0-9]+),
+            "creDate": REGEX([0-9]+\.[0-9]+),
```

Comment:
I understand this time we cannot use `[0-9][0-9][0-9]` or `\d\d\d` instead of `[0-9]+` or `\d+` because in the DB due to rounding effect more than 3 decimal digits may appear, isn't it?

Reply:
I guess we could do something better, I just copied the same approach that was already in the functest - there are many different approaches in the functests and I tried to always use the same one

Reply:
Good enough. NTC